base/scss/overrides/reset/_civicrm.scss

```diff
@@ -121,3 +121,7 @@ input.crm-form-text {
   height: 18px;
   width: 13px;
 }
+
```

Comment:
Could you please add a comment what it does since this is a resetting css file.

Reply:
I also added a missing comment for order arrows.